docs/chapters/baseplatform/obtaining-a-PELUX-release.rst

```diff
@@ -14,20 +14,29 @@ Variables:
 
 Create a directory for the PELUX build. Instruct repo tool to fetch a manifest using the command `repo init`. In this context, branch denotes what branch of git repo `pelux-manifests` to use. Then make repo tool fetch all sources using the command `repo sync`.
 
-.. literalinclude:: snippets/repo-init.sh
-    :language: bash
+.. code-block:: bash
 
-When done fetching the sources, create a build directory and set up bitbake. TEMPLATECONF tells the `oe-init-build-env` script which path to fetch configuration samples from. Note that the example below get the template configuration for the Intel BSP, adapt the path according to your current BSP.
+    mkdir pelux
+    cd pelux
+    repo init -u https://github.com/Pelagicore/pelux-manifests.git -m <manifest> -b <branch>
+    repo sync
 
-.. literalinclude:: snippets/repo-source.sh
-    :language: bash
+When done fetching the sources, create a build directory and set up bitbake. TEMPLATECONF tells the `oe-init-build-env` script which path to fetch configuration samples from.
 
-The script will create configs if there are no configs present, a message about created `conf/local.conf` and `conf/bblayers.conf` files is normal.
+.. note:: The example below get the template configuration for the Intel BSP, adapt the path according to your current BSP.
+
+.. code-block:: bash
+
+    TEMPLATECONF=`pwd`/sources/meta-pelux-bsp-intel/conf/ source sources/poky/oe-init-build-env build
 
+The script will create configs if there are no configs present, a message about created `conf/local.conf` and `conf/bblayers.conf` files is normal.
```

Comment:
You reviewed changes from other PR that this PR is based on and that PR is already merged, so I'll add additional patch and when i push the weird situation of having other PR's patches here would also be automatically resolved. Github should have some infra for a PR being based on another one, or does it already?